### ISSUE TITLE
预训练出错！麻烦解答一下

### ISSUE THREAD

**Issue by lily159** [Walleclipse/Deep_Speaker-speaker_recognition_system#72]
运行pretraining.py时，出现Could not compute output Tensor("softmax_layer/Softmax:0", shape=(None, 3), dtype=float32)错误，请问是没有加model.fit()的问题吗，如果要加，应该怎么加，能完整的写出应该加的代码吗
